rct229/ruletest_engine/ruletest_jsons/section6/rule_6_2.json

```diff
@@ -0,0 +1,274 @@
+        "expected_rule_outcome": "fail",
+        "standard": {
+            "rule_id": "6-2",
+            "ruleset_reference": "(previously 6-2)Table G3.1 (6)(e)",
+            "rule_description": "Interior lighting power for spaces in proposed building with hardwired lighting, including hotel/motel guest rooms and dormitory living quarters shall be >= the value prescribed by Standard 90.1-2019 in Table 9.6.1.\nInterior lighting power for dwelling units shall be >=0.60 W/ft2",
+            "applicable_rmr": "Proposed RMR",
+            "rule_assertion": "<=",
+            "comparison_value": "Expected Value",
+            "rule_dependency": "None",
+            "mandatory_rule": "Yes",
+            "schema_version": "0.0.12"
+        },
+        "rmr_transformations": {
+            "proposed": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Building Open Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "volume": 311.48531251199995,
+                                                "thermostat_cooling_setpoint_schedule": "Cooling Setpoint Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Heating Setpoint Schedule 1",
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "floor_area": 85.47079679999999,
+                                                        "occupant_multiplier_schedule": "Occupancy Schedule 1",
+                                                        "interior_lighting": [
+                                                            {
+                                                                "id": "Space 1 Lighting 1",
+                                                                "lighting_multiplier_schedule": "Space 1 Lighting Schedule 1",
+                                                                "power_per_area": 1.0763910416709723
+                                                            },
```

Comment:
For failed cases, we will need the sum of spaces power per area greater than 11.8 W/m2